### PR TITLE
Create factory web driver initializer

### PR DESCRIPTION
**Pull request by Modgens**

### CODE REVIEW

**Review by MrMarkutan — CHANGES_REQUESTED** [review]
I think we need a factory here (according to the card #21 on trello board)

File: src/main/java/executor/service/utils/WebDriverInitializer.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import java.time.Duration;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class WebDriverInitializer {
+    private static final Logger LOGGER = Logger.getLogger(WebDriverInitializer.class.getName());
+
+    public WebDriver init(WebDriverConfig webDriverConfig, ProxyConfigHolder proxyConfigHolder) {
```
Comment (MrMarkutan) [inline src/main/java/executor/service/utils/WebDriverInitializer.java:18]: where do we get WebDriverConfig? we need to parse it from properties. And ProxyConfigHolder comes from ProxySourcesClient.getProxy(). Correct me, if I am mistaken.

File: target/classes/executorService.properties
[the remark this answers is not in the document]
Comment (MrMarkutan) [reply target/classes/executorService.properties:1]: added in this pr d906975c696719900f7361878d2d25b0dceea1b2

File: src/main/java/executor/service/utils/WebDriverInitializer.java (hunk 1 above)
Comment (MrMarkutan) [reply src/main/java/executor/service/utils/WebDriverInitializer.java:18]: Or should DI factory parse configs?

File: src/main/java/executor/service/utils/WebDriverConfigExecutor.java [hunk 2]
```diff
@@ -0,0 +1,2 @@
+package executor.service.utils;public class WebDriverConfigExecutor {
+}
```
Comment (DenysKravets) [inline src/main/java/executor/service/utils/WebDriverConfigExecutor.java:2]: Class contents are empty, implementation is probably still on your local machine.

**Review by DenysKravets — CHANGES_REQUESTED** [review]

**Pull request merged**